test/jdk/jdk/nio/zipfs/TestPosix.java

```diff
@@ -759,9 +754,19 @@ public void preserveRemainingBits() throws IOException {
         // Sanity check that 'external file attributes' has the expected value
         verifyExternalFileAttribute(zip, expectedBits);
 
-        // Exercise Files.setPosixFilePermissions without changing permissions
+
         Path zipFile = Path.of("preserve-symlink.zip");
         Files.write(zipFile, zip);
+
+        // Verify that a read/synch roundtrip preserves the external file attributes
```

Comment:
typo 'synch'

Reply:
This comment was replaced with "Run the provided action on the ZipFileSystem" following the rewrite described below.